class PhoneNumberDoesNotExist(Exception):
    pass


class AlreadyExistsError(Exception):
    pass
